src/central-server/admin-service/ui/src/plugins/i18n.ts

```diff
@@ -25,18 +25,17 @@
  * THE SOFTWARE.
  */
 import { prepareI18n } from '@niis/shared-ui';
+import enMessages from '@/locales/en.json';
 
-export const availableLanguages = ['en', 'es', 'ru', 'tk'];
+export const availableLanguages = ['en', 'es', 'ru', 'tk', 'pt']; // adicionado suporte para pt (Português do Brasil)
```

Comment:
Please use `pt-BR`, we are adding support for regions too

```suggestion
export const availableLanguages = ['en', 'es', 'ru', 'tk', 'pt-BR']; // adicionado suporte para pt (Português do Brasil)
```

Reply:
Hi, thanks for the feedback!

I will make the requested changes (renaming to pt-BR, rebasing, and updating translations) in the next few days.

If I run into any issues or have questions, I’ll ask here.

Thanks again for reviewing!